dev/io.openliberty.session.6.0.internal/src/io/openliberty/session/SessionData60.java

```diff
@@ -0,0 +1,116 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.session;
+
+import java.util.Enumeration;
+import java.util.Vector;
+import java.util.logging.Level;
+
+import com.ibm.ws.session.SessionContext;
+import com.ibm.ws.session.utils.LoggingUtil;
+import com.ibm.ws.util.ArrayEnumeration;
+import com.ibm.wsspi.session.ISession;
+
+import io.openliberty.session.http.HttpSessionImpl60;
+import jakarta.servlet.ServletContext;
+
+/*
+ * The WAS Http Session adaptation.  Extends the core HttpSessionImpl and adds WAS-specific function
+ *
+ * Since Servlet 6.0
+ */
+public class SessionData60 extends HttpSessionImpl60 {
+
+    private static final long serialVersionUID = -76305717244905946L;
+    protected HttpSessionFacade60 _httpSessionFacade;
+
+    public SessionData60(ISession session, SessionContext sessCtx, ServletContext servCtx) {
+        super(session);
+        _httpSessionFacade = returnFacade();
+        _sessCtx = sessCtx;
+        appName = _sessCtx.getAppName();
+        setServletContext(servCtx);
+
+        if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled() && LoggingUtil.SESSION_LOGGER_CORE.isLoggable(Level.FINE)) {
+            LoggingUtil.SESSION_LOGGER_CORE.log(Level.FINE, "SessionData60" + "Constructor");
+        }
+    }
+
+    protected HttpSessionFacade60 returnFacade() {
+        return new HttpSessionFacade60(this);
+    }
+
+    /*
+     * @see javax.servlet.http.HttpSession#getValue(java.lang.String)
+     */
+    @Override
```

Comment:
Why are getValue, putValue and removeValue on this implementation?  Should getValueNames be made private?